api/src/main/java/org/openmrs/module/dbevent/monitoring/DbEventListenerStatus.java

```diff
@@ -0,0 +1,69 @@
+package org.openmrs.module.dbevent.monitoring;
+
+import lombok.Data;
+import org.openmrs.module.dbevent.DbEvent;
+import org.openmrs.module.dbevent.DbEventListener;
+import org.openmrs.module.dbevent.Operation;
+
+/**
+ * MBean interface to support jmx monitoring of a DbEventListener
+ */
+@Data
+public class DbEventListenerStatus implements DbEventListenerStatusMXBean {
```

Comment:
This class and the interface it implements is here to both provide a high-level status that is available on the listener itself to query, as well as to expose this through JMX so that one could monitor the status of each listener via a monitoring tool.  jconsole works for testing.